spec/features/comments/user_can_comment_on_a_job_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+describe "user sees the comment section on the job page" do
+  scenario "a user is able to add a comment to a job" do
```

Comment:
what happens when a user enters a comment that already exists or is missing required info? be sure to have a test for that